Task3.py

```diff
@@ -0,0 +1,30 @@
+# Multi-paradigm programming languages, Task №3
+# Andrey Fortus IKM-221a
+import math
+
+GENERAL_INFO = 'Multi-paradigm programming languages, Task №3'
+STUDENT_INFO = 'Andrey Fortus IKM-221a'
+TEMPLATE = 'Input {} value: '
+
+# info
+print(GENERAL_INFO)
+print(STUDENT_INFO)
+
+# Math
+while True:
```

Comment:
I'm not sure, but it looks like now you are going to face **break** and stop at the first iteration in any case.
Could you please check if I'm wrong?

Reply:
![image_2023-03-13_22-49-27](https://user-images.githubusercontent.com/113716040/224829022-8f3a1182-f899-4a6c-bf4a-26de4190fb28.png)